work-item-comments_test.go

```diff
@@ -34,6 +35,8 @@ type TestCommentREST struct {
 }
 
 func TestRunCommentREST(t *testing.T) {
+	os.Setenv("ALMIGHTY_CONFIG_FILE_PATH", "../config.yaml")
```

Comment:
Yeah correct, this line is no longer needed actually, forgot to remove , hehe. Thanks!